### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by drdamour**
for #179

### CODE REVIEW

**Comment by drdamour** [comment]
apparently jade has been renamed pug, fyi

File: README.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ An Expressjs generator for Yeoman, based on the express command line tool.
...
-  - Jade
-  - Handlebars
-  - Swig
-  - EJS
-  - Marko
-  - Nunjucks
+  - [Jade](http://jade-lang.com/)
+  - [Handlebars](http://handlebarsjs.com/)
+  - [Swig](http://paularmstrong.github.io/swig/)
+  - [EJS](http://www.embeddedjs.com/)
```
Comment (petecoop) [inline README.md]: http://ejs.co/

**Comment by petecoop** [comment]
Hey could you update the Jade reference to http://ejs.co then it's good to merge?

**Comment by petecoop** [comment]
I've changed this and merged

**Pull request closed without merging**